project/src/components/user-review/user-review.tsx

```diff
@@ -0,0 +1,58 @@
+import { Fragment } from 'react';
+import { Reviews } from '../../types/reviews';
+import { getCountStars, getFormatDate } from '../utils/utils';
+
+type UserReviewProps = {
+    reviews: Reviews;
+  }
+
+function UserReview({ reviews }: UserReviewProps): JSX.Element {
+  return (
+    <Fragment>
+      {reviews.map((review) => {
+        const countStars = getCountStars(review.rating);
+        const date = getFormatDate(review.date);
+        return (
+          < li
```

Comment:
пробелов тут быть не должно, посмотри настройки линтера. он должен такое находить

```suggestion
          <li
```